README.md

```diff
@@ -1 +1,51 @@
-# Pulsar Node Client
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+# Pulsar Node.js client library
+
+The Pulsar Node.js client can be used to create Pulsar producers and consumers in Node.js.
+
+## Requirements
+
+Pulsar Node.js client library is based on the C++ client library. Follow the instructions for
```

Comment:
I would suggest add a section `Compatibility`. So people would have an expectation on what eversion of this node js client was developed.

E.g. `This NodeJS client is developed and tested using Apache Pulsar Client x.y.z`.

Reply:
I still believe we should aim for a single release version. Otherwise things will get confusing on multiple fronts. Eg. from ASF perspective there's only one project "pulsar", hence I don't know how "partial" releases would work there.

Reply:
I don't suggest us doing partial releases here. my suggestion here is to have a section for the current state of master in the repo. when people go to this repo and fork it, he should be aware of what version of Pulsar to use.

Reply:
I also don't suggest the partial releases.
By the way, if we don't do the partial release, should we set the version number of node.js client to same as apache/pulsar? (like 2.3.0)

Reply:
> should we set the version number of node.js client to same as apache/pulsar? (like 2.3.0)

yes. I think we should do that.

Reply:
We added `Compatibility` section and set the version number of node.js client to same as `apache/pulsar`

Reply:
Maybe we can use the same Maven convention for versioning? eg: 2.4.0-SNAPSHOT

Reply:
We can use like eg. set 2.4.0-SNAPSHOT.